package.json

```diff
@@ -25,7 +25,7 @@
     "sn-scatter-plot-ext"
   ],
   "engines": {
-    "node": ">=v18.16.0"
+    "node": ">=v18.15.0"
   },
```

Comment:
or should we fully remove required node version?

Reply:
Either that, or tell renovate not to update engines...